.travis.yml

```diff
@@ -23,3 +23,28 @@ script:
       docker exec -ti temp-ps rm -rf /var/www/html/modules/ps_themecusto;
       docker run --rm --volumes-from temp-ps -v $PWD:/var/www/html/modules/ps_themecusto -e _PS_ROOT_DIR_=/var/www/html --workdir=/var/www/html/modules/ps_themecusto quetzacoalt/phpstan analyse --configuration=/var/www/html/modules/ps_themecusto/tests/phpstan/phpstan.neon;
     fi
+
+before_deploy:
+  # Composer
+  - composer install --no-dev -o
+  # Cleanup
+  - rm -f .php_cs.*
+  - rm -f .travis.yml
+  - rm -f composer.*
+  - rm -f .gitignore
+  - rm -rf .git
+  - rm -rf tests
+  - mkdir -p /tmp/build/ps_themecusto
+  - mv * /tmp/build/ps_themecusto
+  - pushd /tmp/build
+  - zip /tmp/build/ps_themecusto.zip -r ps_themecusto/
```

Comment:
Please check again the original module, we removed the `mv` command as it does not move the hidden files.

Reply:
done